### PR TITLE
Add targetSegmentSizeSecs to transcode API

### PR DESCRIPTION
**Pull request by thomshutt**
- Add target_segment_size_secs to transcode API
- snake_case

**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional)**

**Checklist**

- [x] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/api/src/schema/schema.yaml [hunk 1]
```diff
@@ -1633,6 +1633,9 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/ffmpeg-profile"
+        target_segment_size_secs:
```
Comment (victorges) [inline packages/api/src/schema/schema.yaml]: We should use `camelCase` in the API for consistency. I'm kinda convinced on underscore for the future, but we have only camel field names today so we should keep that. 

(it is a mess for "enums" here tho, we have all of them – camel, kebab, snake... lol)

File: packages/api/src/schema/schema.yaml [hunk 2]
```diff
@@ -1775,6 +1778,9 @@ components:
                   type: array
                   items:
                     $ref: "#/components/schemas/ffmpeg-profile"
+                target_segment_size_secs:
```
Comment (victorges) [inline packages/api/src/schema/schema.yaml]: Same for camel 🐫

**Review by victorges — APPROVED** [review]
LGTM but 🐍➡️🐫

File: packages/api/src/schema/schema.yaml (hunk 1 above)
Comment (thomshutt) [reply packages/api/src/schema/schema.yaml]: Sounds good, I wasn't sure which to go with from looking at the existing fields - will change now

**Comment by victorges** [comment]
Oh also, the build/tests are failing for some reason. Please fix those before merging

**Pull request merged**